features/bevragen/persoon/ouder/autorisatie/autorisatie.feature

```diff
@@ -0,0 +1,291 @@
+      | Rubrieknummer ad hoc (35.95.60) | Medium ad hoc (35.95.67) | Datum ingang (35.99.98) |
+      | <ad hoc rubrieken>              | N                        | 20201128                |
+      En de geauthenticeerde consumer heeft de volgende 'claim' gegevens
+      | naam         | waarde |
+      | afnemerID    | 000008 |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000024                       |
+      | fields              | ouders.ouderAanduiding          |
+      Dan heeft de response 1 persoon
+
+      Voorbeelden:
+      | ad hoc rubrieken               | autorisatie                                                  |
+      | 10120 20120 30120              | exact 1 gegeven uit groep 01                                 |
+      | 10120 20240 30240              | exact 1 gegeven uit groep 02                                 |
+      | 10120 20310 30310              | exact 1 gegeven uit groep 03                                 |
+      | 10120 20410 30410              | exact 1 gegeven uit groep 04                                 |
+      | 10120 26210 36210              | exact 1 gegeven uit groep 62                                 |
+      | 10120 20110 30110              | A-nummer uit groep 01 dat niet voorkomt in de resource       |
+      | 10120 20310 30210              | exact 1 gegeven van ouder 1 en 1 ander gegeven van ouder 2   |
+      | 10120 20120 20240 30310 306210 | meerdere gegevens van ouder 1 en andere gegevens van ouder 2 |
+      | PAOU01                         | alleen voorletters van ouder(s)                              |
+
+      
+  Rule: Vragen met fields om een gegeven van ouders, wanneer de afnemer niet geautoriseerd is voor dan gegeven van ouder 1 én van ouder 2, geeft een foutmelding
+
+    Abstract Scenario: Afnemer vraagt ouders.<fields> en heeft uitsluitend de autorisatie die nodig is om deze vraag te mogen stellen
+      Gegeven de afnemer met indicatie '000008' heeft de volgende 'autorisatie' gegevens
+      | Rubrieknummer ad hoc (35.95.60) | Medium ad hoc (35.95.67) | Datum ingang (35.99.98) |
+      | 10120 <ad hoc rubrieken>        | N                        | 20201128                |
+      En de geauthenticeerde consumer heeft de volgende 'claim' gegevens
+      | naam         | waarde |
+      | afnemerID    | 000008 |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000024                       |
+      | fields              | ouders.<fields>                 |
+      Dan heeft de response 1 persoon
+
+      Voorbeelden:
+      | ad hoc rubrieken                                       | fields                                              |
```

Comment:
missen we niet een scenario voor welke rubrieken een afnemer moet zijn geautoriseerd als hij met fields 'ouders' bevraagt?

Reply:
er is ook geen scenario voor welke rubrieken een afnemer moet zijn geautoriseerd bij vragen van fields met ouders.ouderAanduiding

Reply:
scenario toegevoegd met minimale autorisatie voor vragen hele groep ouders

Reply:
voor ouderAanduiding is er wel een scenario, toch? Op regel 42. Die is niet 100% dekkend, omdat er enorm veel combinaties mogelijk zijn, maar het principe wordt dacht ik wel getoond.
Dit is trouwens niet nieuw, alleen de mogelijkheid van PAOU01 is toegevoegd.